addons/stock_account/models/stock_valuation_layer.py

```diff
@@ -13,7 +13,6 @@ class StockValuationLayer(models.Model):
 
     _rec_name = 'product_id'
 
-    active = fields.Boolean(related='product_id.active')
```

Comment:
you can now remove this patch 466a8319547e11e2193dd8594b0e50e3c27d4b3e and we'll be good i think